projects/addon-doc/src/services/theme-night.service.ts

```diff
@@ -0,0 +1,38 @@
+import {Inject, Injectable} from '@angular/core';
+import {LOCAL_STORAGE, WINDOW} from '@ng-web-apis/common';
+import {BehaviorSubject, Subject} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+import {TUI_THEME_NIGHT_STORAGE_KEY} from './theme-night.options';
+
+@Injectable({
+    providedIn: `root`,
+})
+export class TuiThemeNightService extends BehaviorSubject<boolean> {
+    readonly change$ = this.pipe<void>(map(() => undefined)) as Subject<void>;
```

Comment:
Why `Subject`? And why do we need it at all?

Reply:
fixed